docs/release/1.5/release-note-1.5.0.md

```diff
@@ -0,0 +1,15 @@
+# Release Note 1.5.0
+
+## Feature Highlights
+
+- `ONE Compiler`
```

Comment:
@lemmaa created several PR to change `ONE` or _ONE_ to **ONE**. Maybe we need to use "**ONE** compiler"?

Reply:
I've heard not answer but it was marked _resolved_.
Sorry but I worked in marketing before so I am obsessed with naming and how it is shown to public.

Reply:
This is my mistake. I will update.

Reply:
@hyunsik-yoon I have bolded ONE word.